packages/Modal/boolean/src/ModalBoolean.spec.tsx

```diff
@@ -1,10 +1,11 @@
 import * as React from 'react';
-import { mount } from 'enzyme';
+import { render, screen } from '@testing-library/react';
+import UserEvent from '@testing-library/user-event';
 import BooleanModal from './ModalBoolean';
 
 describe('<BooleanModal>', () => {
   it('should render boolean modal', () => {
-    const modal = mount(
+    const { asFragment } = render(
```

Comment:
Généralement je test mes snapshot en destructurant container au lieu de fragment et en vérifiant comme ca : 

expect(container.firstChild).toMatchSnapshot();

C'est quoi la diff ?

Reply:
Aucune quasiement, ici aucune ça renvoie la même chose. J'ai pris l'habitude d'utiliser toujours ca

Le asFragment va te retourner un DocumentFragment de toute ton app ou d'une partie (que tu mets en paramètre). 

A la seule différence près que ca va choper le fragment au moment ou tu l'appelle. 

Exemple : 

```javascript
const TestComponent = () => {
      const [count, setCounter] = useState(0);
      return <button onClick={() => setCounter(count => count + 1)}>Incrémente: {count}</button>;
    };

    const { getByText, asFragment, container } = render(<TestComponent />);

    fireEvent.click(getByText(/Incrémente/));

    // This will snapshot only the difference between the first render, and the
    // state of the DOM after the click event.
    // See https://github.com/jest-community/snapshot-diff
    expect(container.firstChild).toMatchInlineSnapshot(`
      <button>
        Incrémente: 
        1
      </button>
    `);
    expect(asFragment()).toMatchInlineSnapshot(`
      <DocumentFragment>
        <button>
          Incrémente: 1
        </button>
      </DocumentFragment>
    `);
```